### ISSUE TITLE
Solr disappeared or moved ?

### ISSUE THREAD

**Issue by ChristopheLRTE** [TOSIT-IO/tdp-getting-started#175]

**Comment by leopaul36** [comment]
Hello @ChristopheLRTE ,

Solr installation tasks are still available at https://github.com/TOSIT-IO/tdp-collection/tree/master/roles/ranger/solr

I moved the issue because it is related to tdp-getting-started. No sure when or why but indeed the Solr installation part was removed from https://github.com/TOSIT-IO/tdp-getting-started/blob/master/deploy-ranger.

Feel free to open a PR :)

**Comment by leopaul36** [comment]
I think our best option is to include Solr to the the tdp-lib dag at https://github.com/TOSIT-IO/tdp-collection/blob/master/tdp_lib_dag/ranger.yml and regenerate the meta playbooks is that correct @rpignolet ?

**Comment by rpignolet** [comment]
Yes @leopaul36 we should have ranger solr in the DAG and it is located inside the `tdp-collection` repository.

**Comment by leopaul36** [comment]
Moving back to `tdp-collection` as the DAG is there
